src/components/Icon/index.js

```diff
@@ -40,6 +41,9 @@ const Icon = ({ name, size, className, src, button, children, style, sizePx, ...
       {...props}
       className={cx(styles.container, styles[size], className, button && styles.button)}
       style={buildStyle({ style, src, name, imageSrc, sizePx })}
+      role="button"
```

Comment:
semantically it doesn't make sense if we have this as `role=button` for all icons.

can we conditionally set this if onClick is passed in? (as well as potentially add some tests so coverage passes? :) )

Reply:
Yeah, you're right, it shouldn't have a button role in that case... I originally tried to fix the `jsx-a11y/no-static-element-interactions` problem by increasing the eslint version, in which case the `div` was allowed an `onClick` handler if it also included a `role` and a `tabIndex`. The version increase brought some other breaking changes, so I didn't want to include that in the pull request, but I kept those attributes. I'll change it so that it's a role-less `<div>` without click handling, and an `<a>` with one. The role is a bit tricky - I will be using it as a `radio` on my website, but that won't be enforced by this library. However, I'm not sure if it's worth propagating in the props, though.

Regarding the coverage: how exactly is the Codecov pass determined? Passing some relative threshold, or a positive delta? I had some issues with the test scripts, mostly since the `unit` and `coverage` scripts are platform-dependent and don't work on Windows. I tried updating them temporarily to run the tests before commiting but there still appear to be some configuration issues with babel... Now I see there's also a `codecov` script, but I didn't manage to get it working yet either. This being said, I wouldn't mind adding tests for my changes, but I wouldn't want to spend too much time on adding tests for existing code or making the scripts work under Windows, after all I also need to work on my website sometimes :P I might end up trying to add something to the Trait spec tests, but I probably won't go too overboard.

Reply:
sounds good.

re: coverage, it just errors if coverage goes down. so if we can get coverage to the same threshold then we're golden!